test/acceptance/member_test.exs

```diff
@@ -18,19 +18,4 @@ defmodule Community.Acceptance.MemberTest do
     assert last_member.company_name == "X-men"
     assert last_member.approved == false
   end
-
-  defp click_on(text) do
-    find_element(:link_text, text)
-    |> click
-  end
-
-  defp click_role(name) do
```

Comment:
I'm not sure about extracting click_role. Since it's only used in this test, it might be better to keep it here so it's easy to see what it does, and change it if necessary. The other file seems like it's more for generic feature helpers. If you ever do need click_role in multiple places, it might be more clear to put it in a `RoleFeatureHelper` that you alias or import only in the tests that need it